en/security.rst

```diff
@@ -18,8 +18,298 @@ ACL (Access Control List)
+
+.. code-block:: bash
+
+	$ cubrid tde --show-keys testdb
+	Key File: /home/usr/CUBRID/databases/testdb/testdb_keys
+
+	The current key set on testdb:
+	Key Index: 2
+	Created on Fri Nov 27 11:14:54 2020
+	Set     on Fri Nov 27 11:15:30 2020
+
+	Keys Information: 
+	Key Index: 0 created on Fri Nov 27 11:11:27 2020
+	Key Index: 1 created on Fri Nov 27 11:14:47 2020
+	Key Index: 2 created on Fri Nov 27 11:14:54 2020
+	Key Index: 3 created on Fri Nov 27 11:14:55 2020
+
+	The number of keys: 4
+
+.. note:: **Creating a database using an existing key file**
+
+    If you want multiple databases to share an existing or new key file, you can create a database after setting the path of the key file by using the **tde_keys_file_path** system parameter. For more information on the system parameters, see :ref:`other-parameters`.
+
+.. _tde-enc-target:
+
+Encryption Target
+-----------------
+
+.. _tde-enc-perm:
+
+Permanent Data Encryption
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The encrypted table data and all index data created on the table are encrypted. For more information on the encrypted table, see :ref:`create-tde-table`.
+
+.. _tde-enc-temp:
+
+Temporary Data Encryption
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In addition to persistent data such as tables, temporary data created during queries related to encrypted tables are also encrypted. For example, all temporary data created in the process of executing a query such as `SELECT * FROM tde_tbl ORDER BY att1` or creating an index on `tde_tbl` are encrypted when it is written to disk. For more information on temporary data, see :ref:`temporary-volumes`.
+
+.. _tde-enc-log:
+
+Log Data Encryption
+^^^^^^^^^^^^^^^^^^^
+
+Since the data which has to be encrypted may be included in the REDO and UNDO log records generated when the encrypted table is manipulated, all log data related to the encrypted table is encrypted. Encryption is applied to both the active log and the archive log. For more information on log volumes, see :ref:`database-volume`.
+
+.. _tde-enc-dwb:
+
+DWB Encryption
+^^^^^^^^^^^^^^
+
+Persistent data is temporarily written to the Double Write Buffer (DWB) before being written to the data volume. Even at this time, it may be encrypted because the data for the encrypted table can be included. For more information on DWB, see :ref:`database-volume`.
```

Comment:
```suggestion
Persistent data is temporarily written to the Double Write Buffer (DWB) before being written to the data volume. It may be encrypted even at this time because the data for the encrypted table can be included. For more information on DWB, see :ref:`database-volume`.
```